flow/src/main/java/flow/Flow.java

```diff
@@ -146,8 +146,8 @@ public void goTo(final Path path) {
   }
 
   /**
-   * Reset to the specified screen. Pops until the screen is found. If the screen is not found,
-   * the entire backstack is replaced with the screen.
+   * Set the specified screen to the top of the backstack. If the screen already
+   * exists in the backstack, the screen will be moved to the top of the backstack.
```

Comment:
@rjrjr Rereading that, I agree, it is misleading. Thanks for taking the time to review it. Since it looks  like this code block is possibly changing anyways, (issue #55) this PR can be removed.